src/common/gcode/gcode_info.cpp

```diff
@@ -497,6 +497,13 @@ void GCodeInfo::parse_gcode(GcodeBuffer::String cmd, uint32_t &gcode_counter) {
     if (cmd.front() == ';' || cmd.is_empty()) {
         return;
     }
+
+    // Make sure we don't parse inline comments
+    auto equal = std::find(cmd.begin, cmd.end, ';');
```

Comment:
Why `equal`? What does it say to the reader?

Reply:
changed it to comment_line

Reply:
It's now only "comment" :)